appveyor.yml

```diff
@@ -12,10 +12,15 @@ branches:
   only:
     - master
     - /^release-.*/
+    - /^cv.*/
 
+# Note: use `[ci skip]` or `[skip ci]` in the commit message title and AppVeyor won't be
+# built for that commit. You can use `[skip appveyor]` to explicitly skip AppVeyor and
+# allow other CI to still run.
 skip_commits:
-# Add [av skip] to commit messages for docfixes, etc to reduce load on queue
-  message: /\[av skip\]/
```

Comment:
I need to run another test to validate this removal is necessary as originally I messed up the `files` rule.